services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api/_utils.py

```diff
@@ -0,0 +1,24 @@
+from contextlib import asynccontextmanager
+from typing import AsyncIterator
+
+import aiodocker
+
+from ..errors import GenericDockerError
+
+
+class _RetryError(Exception):
```

Comment:
what is this exception used for?

Reply:
It's used in tenacity to trigger a retry if certain conditions occur.

Reply:
But, this is a protected class and I cannot find it anywhere in the module ``_utils.py``